modules/customize-certificates-api-update-named.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="customize-certificates-api-update-named_{context}"]
+= Updating an API server named certificate
+
+You can update the API server certificate to a new version by creating a new secret and updating the API server.
+
+.Procedure
+
+. Log in to the API as the `kubeadmin` user.
++
+[source,terminal]
+----
+$ oc login -u kubeadmin -p <password> https://FQDN:6443
+----
+
+. Create a secret with a new name in the openshift-config` namespace that contains the certificate chain and private key.
```

Comment:
Missing the backtick before "openshift-config" (sorry, that was my fault - my suggestion was missing it)
```suggestion
. Create a secret with a new name in the `openshift-config` namespace that contains the certificate chain and private key.
```

Reply:
Done!